gpameter.user.js

```diff
@@ -27,9 +27,15 @@ var grade_values = {
 };
 console.log(studentId);
 add_checkboxes = function(){
+    var mapElems = new Map();
```

Comment:
Please fix the indentation here.
Current code uses tabs, while this line uses spaces.
Let's be consistent.